docs/communicate-with-javascript.md

```diff
@@ -739,7 +739,7 @@ These attributes are used to annotate `external` definitions:
 - [`bs.val`](#bind-to-global-javascript-functions-or-values): bind to global
   JavaScript functions or other values
 - [`bs.variadic`](#variadic-function-arguments): bind to a function taking
-  variadic arguments from an array
+  variadic arguments from an array (`bs.variadic` has replaced `bs.splice`, which is now considered deprecated; `bs.splice` still exists and can be used for backwards compatibility)
```

Comment:
As the attributes are ordered alphabetically for easier finding, let's leave the variadic entry as it was, and add an new entry for `bs.splice` above, after `bs.scope`:

```
- [`bs.splice`](#variadic-function-arguments): a deprecated attribute, is an
  alternate form of `bs.variadic`
```

Reply:
Okay. I've updated the text.